Extensions/probing.hh

```diff
@@ -281,11 +307,16 @@ inline double getReactivityScore(const Subsequence &leftBase, const bool isUnpai
 		    }
 		    infile.close();
 		}
-		if (probingData.size() < (leftBase.seq->n)) {
-			std::cerr << "Warning: chemical probing data file '" << getProbing_dataFilename() << "' misses " << (leftBase.seq->n - probingData.size()) << " data-row(s) " << std::endl << "         compared to the number of nucleotides in your input sequence." << std::endl << "         Missing values will be set to 0.0!" << std::endl;
+        unsigned int seqLength = leftBase.seq->n + getInput_sequence(1).second + 1;
```

Comment:
This line here is why the core dumps happen. 
If the function is called with a single track input the "getInput_sequence(1).second" doesn't exist, therefore an "index out of range" error is thrown and the whole process exits with an error.

I fixed the issue in my current commit in re-defining the `getInput_sequence` function and testing the length of the `input` vector. 

Either you include this in your version or maybe find some other way to deal with this problem.